ipset_linux.go

```diff
@@ -250,7 +250,7 @@ func (h *Handle) IpsetListAll() ([]IPSetResult, error) {
 		result[i].unserialize(msg)
 	}
 
-	return result, nil
+	return result, err
```

Comment:
no need to check for ErrDumpInterrupted ?

Reply:
Ah, thank you - I think I modified this one as it's a `...ListAll()` - then realised it doesn't set `NLM_F_DUMP`, and didn't fully undo the change. I'll change the return back to `nil`.